s3tests/analysis/rwstats.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
If this code is going to be run in the venv creates as about, there is no sense to use shebang at all.